README.md

```diff
@@ -31,7 +31,8 @@ Visit `[url of project]/messages` and enter the number of the whitelisted phone
 
 We love to hear from you so if you have questions, comments or find a bug in the project, let us know! You can either:
 
-* Open an issue on this repository
+* Open an issue on this repository. 
+* Before you start contributing, check out the [Code of Conduct](CODEOFCONDUCT.md).
```

Comment:
hey @imanishbarnwal this isn't actually a valid URL, can you change it and test so that you know it works?

Reply:
Oh so sorry, thanks for highlighting this I just updated this. Kindly check it now.